ui/src/components/search.js

```diff
@@ -55,11 +55,12 @@ class Search extends React.Component{
     displayChanged=(event)=>{
         this.setState({['display']:event})
     }
+    
 
     load=()=>{
         searchItems(this.searchText,this.selectedCategories,this.loadCount,6,this.sort,(success, data)=>{
             if(success){
-                if(data.length===0){
+                if(data.length<6){
```

Comment:
extract to constants as named variable

Reply:
also, there is a bug here, if we have 12 items, we fetch page one, button is not disabled, we fetch page 2, button it still not disabled (but it should be)